packages/apps/graph/src/devnet/simulation/simulate.ts

```diff
@@ -174,7 +178,7 @@ export async function simulate({
               ...nextAccount,
               chainId: `${getRandomNumber(
                 seededRandomNo,
-                devnetConfig.NUMBER_OF_CHAINS,
+                dotenv.CHAIN_COUNT,
```

Comment:
Maybe not in scope of this PR, but if `CHAIN_COUNT` is set to 0, then this will be an issue, because it'll try to make a cross-chain transfer to same chain accounts

Reply:
I'll add the extra check just in case